packages/dhis2/src/Adaptor.js

```diff
@@ -146,7 +146,7 @@ axios.interceptors.response.use(
  *   orgUnit: 'DiszpKrYNg8',
  *   status: 'COMPLETED',
  * });
- * @example <caption>a trackedEntityInstance</caption>
+ * @example <caption>a trackedEntityInstance. Create a tracker instance. See {@link https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-241/tracker.html#webapi_nti_import Create tracker docs}</caption>
```

Comment:
In the example, I would call it a `trackedEntity`.

I'm starting to think we should drop support for the term `trackedEntityInstance`.

I'm not very good at DHIS2 but is  "Create a tracker instance" the right language? I don't see that language in the DHIS2 API docs. I suspect you're not creating a tracker, I think you're creating an entity